contents/blog/remote-culture.md

```diff
@@ -0,0 +1,238 @@
+
+At PostHog, for example, we don't send internal emails. 99% of the time, we're talking on GitHub, Slack, or a call. We also mostly use Slack for non-work-related, private, and more urgent conversations. If it doesn't fit those categories, chances are it could be on GitHub.
+
+#### Transparency
+Transparency means that if some information or a particular conversation can be public, we should try to make it public. 
+
+This way, we create an ever-growing record of things that have been previously discussed, questions that have been asked, and answers that have been found. 
+
+As such, when someone needs some info, they don't need to message a team member to get what they need. They can just find it immediately. 
+We extend this concept quite deeply, having our entire [Company Handbook](/handbook) public, which outlines things like how we do sales calls, our approach to talking to investors, and how we hire people. As such, if there's ever a question about how we operate as a company, referring to the handbook will usually get you the answer. 
+
+Additionally, our massive backlog of issues and pull requests also contains very useful information regarding how we've come to certain decisions, from engineering to marketing. 
+
+#### Written Communication
+Underpinning the aforementioned points is our preference for written communication. 
+
+At PostHog, a large part of our async and transparent culture is underpinned by written communication, so I'll keep this section short. It's worth briefly mentioning a few points about how written communication helps us, however:
+
+* **Faster:** If something doesn't need to be a call, it shouldn't be a call. Calls are great for the social aspect they bring, but for day-to-day work, they often are a slower process for making decisions, given that they include small-talk, internet issues, and scheduling difficulties. Communicating via writing can help save time.
+* **Referenceable:** Unless you record all your calls, chances are you might forget something that was discussed in a meeting. However, by trying to write instead of calling, and taking notes when we do call, we can always refer back to what was discussed and easily expand on the discussion later on.
+* **Clearer:** Writing something out requires some effort to communicate your point of view effectively. As such, writing can help you distill your thoughts into concrete points, as well as present these points to others in a more solid way. 
+
+## Pull Requests > Issues
```

Comment:
This is a good special policy. It's about being autonomous and proactive - we need people to be like this when they on't have the constant interaction/guidance that an in person office would provide